font-patcher

```diff
@@ -688,46 +683,34 @@ class font_patcher:
             if exactEncoding:
                 # use the exact same hex values for the source font as for the symbol font
                 currentSourceFontGlyph = sym_glyph.encoding
-
-                # Save as a hex string without the '0x' prefix
-                copiedToSlot = format(sym_glyph.unicode, 'X')
             else:
                 # use source font defined hex values based on passed in start and end
-                # convince that this string really is a hex:
-                currentSourceFontGlyph = int("0x" + sourceFontList[sourceFontCounter], 16)
-                copiedToSlot = sourceFontList[sourceFontCounter]
+                currentSourceFontGlyph = sourceFontList[sourceFontCounter]
                 sourceFontCounter += 1
 
-            if int(copiedToSlot, 16) < 0:
-                print("Found invalid glyph slot number. Skipping.")
-                continue
-
```

Comment:
`custom` still works which is when this was added. probably safe and unneeded so this seems okay to remove 👍🏻